cache/lru/cache.go

```diff
@@ -55,14 +77,26 @@ func (c *Cache[K, V]) Get(key K) (V, bool) {
 func (c *Cache[K, _]) Evict(key K) {
 	c.lock.Lock()
 	defer c.lock.Unlock()
-
+	if c.onEvict != nil {
+		if value, found := c.elements.Get(key); found {
```

Comment:
nit: Go typically uses `ok` instead of `found` for the var name for these existence checks